src/Pokemon/Attacks.java

```diff
@@ -0,0 +1,7 @@
+package Pokemon;
+
+import java.util.List;
+
+public interface Attacks {
```

Comment:
Deze interface voegt niet heel veel toe, omdat alle pokemons attacks hebben. Maar dat is natuurlijk ook een beetje de scope van de opdracht. Wel een leuke toevoeging en goed dat je hem ook consequent implementeert